docs/tutorials/advanced_tutorials/cloud-submission.md

```diff
@@ -0,0 +1,124 @@
+Moreover, the requested result type needs to be defined.
+These are provided in a list, so that multiple result types can be requested in a single submission.
+
+```python
+from qadence import BackendName
+from qadence.pasqal_cloud_connection import WorkloadSpec, ResultTypes
+
+workload = WorkloadSpec(circuit, BackendName.PYQTORCH, [ResultTypes.SAMPLE, ResultTypes.RUN])
+```
+
+### Using a Quantum Model
+
+If you already have your quantum computation defined as a `QuantumModel`, it is possible to create a workload specification directly from the model using `get_spec_from_model`.
+Then, the circuit and backend specifications will be extracted from the model, the other values need to be provided as extra arguments.
+
+```python
+from qadence.pasqal_cloud_connection import get_spec_from_model
+
+model = QuantumModel(...)
+workload = get_spec_from_model(model, [ResultType.SAMPLE])
+```
+
+### Observable Expectation Value
+
+For the result type `ResultType.EXPECTATION` it is mandatory to provide an observable to the workload specification.
+In the example below we use the trivial identity observable `I(0) @ I(1)`.
+
+```python
+workload = WorkloadSpec(circuit, BackendName.PYQTORCH, [ResultTypes.EXPECTATION], observable=I(0)*I(1))
+```
+
+### Parametric Circuits
+
+In the case of a parametric circuit, _i.e._ a circuit that contains feature parameters or variational parameters, values for these parameters need to be provided.
+The parameter values are defined in a dictionary, where keys are the parameter name and values are parameter value passed as torch tensors.
+The parameter values are defined in a dictionary, where keys are the parameter name and values are parameter value passed as torch tensors.
+It is possible to set multiple values by using a 1-D tensor, to the parameters, in that case the computation is executed for each value in the tensor.
+A mix of 0-D and 1-D tensors can be provided to keep some parameters constant and others changed during this process.
+However, all 1-D tensors need to have the same length.
+
+
+```python
+parametric_circuit = ...
+parameter_values = {"param1": tensor(0), "param2": tensor([0, 1, 2]), "param3": tensor([5, 6, 7])}
+workload = WorkloadSpec(parametric_circuit, BackendName.PYQTORCH, [ResultTypes.EXPECTATION], parameter_values=parameter_values)
+```
+
+## Submission
+Submission to the cloud platform is done very easily using the `submit_workload` function.
+The workload id will be provided by executing the function.
+This id is needed later, to request the status of the given workload.
+
+```python
+from qadence.pasqal_cloud_connection import submit_workload
+
+workload_id = submit_workload(connection, workload)
+```
+
+## Check Workload Status
+
+The `check_status` function can be used to see if the workload is finished already.
+The status of a workload can be: done, pending, running, paused, canceled, timed out or error.
```

Comment:
```suggestion
The status of a workload can be: done, pending, running, paused, canceled, timed out or error.
The `check_status` function can be used to see if the workload is finished already.
```

So that it fits nicely with the next sentence